packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -193,10 +194,15 @@ export default abstract class EasCommand extends Command {
     if (err instanceof EasCommandError) {
       Log.error(err.message);
     } else if (err instanceof CombinedError && err?.graphQLErrors) {
-      let cleanMessage = err.message.replace('[GraphQL] ', '');
-      cleanMessage += err?.graphQLErrors?.[0]?.extensions?.requestId
-        ? `\nRequest ID: ${err.graphQLErrors[0].extensions.requestId}`
-        : '';
+      const cleanMessage = err?.graphQLErrors
+        .map((graphQLError: GraphQLError) => {
+          const messageLine = graphQLError.message.replace('[GraphQL] ', '');
```

Comment:
Previously we removed the `[GraphQL]` part from the `CombinedError` message, not from a single one `GraphQLError`. Did you verify that a single `GraphQLError` can contain this part and if this logic makes sense in this context?

Reply:
From what I saw, the `err.message` property was a concatenation of `err.graphQLErrors.[*].message` properties, therefore I assumed it those messages on graphQLErrors that have this `[GraphQL]` label in the beginning. I may be wrong, but even so would there be a problem with replacing it if it's not there? The `replace()` function wouldn't do anything in that case